### ISSUE TITLE
The confidence  or category probability value is too small

### ISSUE THREAD

**Issue by silicon2006** [ultralytics/yolov5#837]
## ❔Question
When testing, the confidence  or category probability value is too small, even when testing on the training data set, the confidence value is also small. 
However, the confidence and category probability of the darknet version of yolov4 and yolov3 can reach 0.99 (on the training data set)， why?  and how to fix it?

## Additional context
![Snipaste_2020-08-25_10-48-14](https://user-images.githubusercontent.com/13852906/91117172-8ce2bb00-e6c0-11ea-8cf3-786ef80fa603.png)

**Comment by glenn-jocher** [comment]
@silicon2006 we output object confidence times class confidence times predicted IoU.

I think yolov3 and yolov4 are outputing something a bit simpler (and less performant).

**Comment by Aktcob** [comment]
iou aware.  And score = sqrt(pred_iou*cls) can get score with 0.9+++

**Comment by silicon2006** [comment]
Good Idea, Now the socre (object confidence times class probability) can reach 0.95+ when testing on the train dataset;   Thanks!

**Comment by lucasjinreal** [comment]
@Aktcob what's pred_iou when inference?

**Comment by glenn-jocher** [comment]
@jinfagang if training is too short, predicted iou will be lower, since iou is lower. This will result in lower confidences as well, so low confidences during inference is a good indicator that you have not trained long enough.

**Comment by maryamag85** [comment]
I am using default pretrained resnet101 and scores are still too small.

**Comment by Steinwang** [comment]
> @silicon2006 we output object confidence times class confidence times predicted IoU.
> 
> I think yolov3 and yolov4 are outputing something a bit simpler (and less performant).

So how can I cancel this setting? Output objectness confidence score directly without scaled by IOU and class confidence

**Comment by glenn-jocher** [comment]
@Steinwang objectness targets can be modified in the loss function:
https://github.com/ultralytics/yolov5/blob/71621df87589faea19ba4c4098bb68e73201f30c/utils/loss.py#L144-L152
